lib/src/ruby/module.cc

```diff
@@ -327,6 +330,47 @@ namespace facter { namespace ruby {
         return ruby.to_native<fact>(fact_self)->value();
     }
 
+    VALUE module::to_ruby(value const* val) const
```

Comment:
On the other hand, it's better not to call `api::instance` before a `module`'s been created, so let's leave this an instance method.